buildout-mercator.cfg

```diff
@@ -9,6 +9,7 @@ dump-requirements-file = ${make_wheels:platform}-requirements.txt
 frontend.static_dir = src/mercator/mercator/build
 frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
+frontend.profile_images_enabled = false
```

Comment:
Yes, I'll take care. I'm okay with disabling by default. We just had conversations in the past that it'd be good if contributions from external partners should be enabled in order to make liqd aware that they exist, but that can also be done in a separate PR.